src/log-worker/logWorker.ts

```diff
@@ -0,0 +1,21 @@
+import Worker = require('tortoise');
+import { DotenvParseOutput } from 'dotenv';
+
+import { createWorker, Message, consumeMessage } from '.';
+
+const { log, error } = console;
+
+export const logWorker = async (config: DotenvParseOutput): Promise<void> => {
+  const worker: Worker = await createWorker(config).catch(e => error(e));
```

Comment:
If an error was caught in the catch block, worker will be null, how would the consumeMessage function behave 🤔. You might want to return early here.

Reply:
ok. done.